app/src/main/java/com/itsaky/androidide/activities/MainActivity.kt

```diff
@@ -162,9 +167,11 @@ class MainActivity : LimitlessIDEActivity() {
       // template list -> template details
       // ------- OR -------
       // template details -> template list
+      // ------- OR -------
+      // project list -> home
       val setAxisToX =
-        (previous == SCREEN_TEMPLATE_LIST || previous == SCREEN_TEMPLATE_DETAILS) &&
-            (screen == SCREEN_TEMPLATE_LIST || screen == SCREEN_TEMPLATE_DETAILS)
+        (previous == SCREEN_TEMPLATE_LIST || previous == SCREEN_TEMPLATE_DETAILS || previous == SCREEN_PROJECT_RECENT_LIST ) &&
+            (screen == SCREEN_TEMPLATE_LIST || screen == SCREEN_TEMPLATE_DETAILS || screen == SCREEN_PROJECT_RECENT_LIST)
```

Comment:
This is not needed.

The `setAxisToX` is set only for X-axis (horizontal) transitions. Also, the value of `setAxisToX` will always be false for projects list fragment because :

- if `previous` is `SCREEN_PROJECT_RECENT_LIST`, then RHS of the `&&` comparison will always be false (because user can only navigate to the main fragment from the recent projects list fragment).
- if `previous` is `SCREEN_PROJECT_RECENT_LIST`, then `screen` will not be `SCREEN_PROJECT_RECENT_LIST` either (because the user can only navigate to the projects list fragment from the main screen which is not the case in this).

So, the `setAxisToX` will always be false and the fragment will always be transitioned on the Y-axis (which is happening).

Reply:
Also, update the initializer for `isForward` so that it compares with `>= 1` instead of `== 1` to apply appropriate forward/backward transitions while navigating between main and projects list fragment.